guides/plugins/plugins/administration/using-base-components.md

```diff
@@ -0,0 +1,37 @@
+# Using base components
+
+The Shopware 6 Administration registers all components into the component registry wich makes them available in the entire Administration.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/), the framework, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Finding the base component needed
+
+All Shopware 6 Administration components can be seen in the [Component Library](https://component-library.shopware.com/). 
+There you can see what each of the components does and looks like, it also shows you what props the can work with and wich slots they have.
+
+## Using the base
+
+As mentioned before in the introduction, all components used in the Shopware 6 Administration are registered in the `component registry`, which makes them accessible everywhere.
```

Comment:
You are directly contradicting to your sentence before, where you state, that the components are available anywhere in the Administration.

```suggestion
As mentioned before in the introduction, all components used in the Shopware 6 Administration are registered in the `component registry`, which makes them accessible in the Administration.
```